forumone/gesso-wp
# Issue 28 grunt task updates

Updates on this branch will allow for Patternlab auto-reloading in Gesso.

To test, check this branch out and then

1. Install a Patternlab instance
$ composer create-project pattern-lab/edition-drupal-standard pattern-lab

2. Make sure you have all the new node modules installed.
$ npm install

This has been tested to run in tandem with a Vagrant box. This update spins up a separate server for the Patternlab instance.

*Please note that because of a known issue with the grunt-contrib-connect task you must use a full path when defining the pattern-lab location inside of the grunt watch task. A note has been included there.

*This approach should work for the Drupal Gesso theme as well.

Handles #28
